alf/algorithms/mcts_algorithm.py

```diff
@@ -573,28 +596,32 @@ def rollout_step(self, time_step: TimeStep, state: MCTSState):
         else:
             self._build_tree2(trees, to_plays)
 
-        action, info = self._select_action(trees, steps)
+        action_dist, info = self._select_action(trees, steps)
```

Comment:
Suggestoins: call it `action_probs` instead of `action_dist` as it seems to me that the convention here is `action_dist` = an actual distribution object, while `action_probs` = a tensor of probabilities.

Reply:
changed.